src/screens/HyperSwitch/RoutingRevamp/AdvancedRoutingUIUtils.res

```diff
@@ -341,7 +341,7 @@ module RuleFieldBase = {
 
     React.useEffect0(() => {
```

Comment:
remove the useEffect and use the ` Window.getThreeDsKeys` directly

Reply:
Removed use effect and changed it to useMemo instead